### PR TITLE
Feature/fix allowed programmes

### PR DESCRIPTION
**Pull request by Eleonore9**
* Fix the way of getting an id for programme allowed (get one id out of a seq of 1 element)
* Replace map + remove by keep
* replace as-> by destructuring in a let

### CODE REVIEW

File: src/clj/kixi/hecuba/api/programmes.clj [hunk 1]
```diff
@@ -147,9 +146,9 @@
   (::item ctx))
 
 (defn resource-handle-ok [ctx]
-  (api/render-item ctx
-                    (as-> (::item ctx) item
-                          (assoc item :projects (format programme-projects-index (:programme_id item))))))
+  (let [{:keys [::item]} ctx]
```
Comment (sw1nn) [inline src/clj/kixi/hecuba/api/programmes.clj]: Oooh, I like to see a clojure 1.7.0 feature !

File: src/clj/kixi/hecuba/api/programmes.clj [hunk 2]
```diff
@@ -31,13 +31,12 @@
   (let [allowed-programme-ids      (into #{} (keys allowed-programmes))
         allowed-project-ids        (into #{} (keys allowed-projects))]
     (->> all-programmes
-         (map (fn [programme]
+         (keep (fn [programme]
```
Comment (sw1nn) [inline src/clj/kixi/hecuba/api/programmes.clj]: Good use of `keep`, but now the `->>` is redundant

**Comment by sw1nn** [comment]
minor niggles only, :shipit:

File: src/clj/kixi/hecuba/api/programmes.clj (hunk 1 above)
Comment (Eleonore9) [inline src/clj/kixi/hecuba/api/programmes.clj]: :sparkles: :new: :sparkles:

File: src/clj/kixi/hecuba/api/programmes.clj (hunk 2 above)
Comment (Eleonore9) [inline src/clj/kixi/hecuba/api/programmes.clj]: That should be fixed now

**Pull request merged**